public/client/TracyProfiler.cpp

```diff
@@ -1677,6 +1712,10 @@ void Profiler::Worker()
     new(m_broadcast) UdpBroadcast();
 #  ifdef TRACY_ONLY_LOCALHOST
     const char* addr = "127.255.255.255";
+#  elif defined __QNX__
+     // global broadcast address of 255.255.255.255 is not well-supported by QNX,
+     // use the interface broadcast address instead, e.g. "const char* addr = 192.168.1.255;"
+#    error Need to set an appropriate broadcast address for a QNX target.
```

Comment:
Basing this on an ancient forum post here: https://forums.openqnx.com/t/topic/4018 and the fact that the limited broadcast address failed for me and the interface broadcast addressed worked.